lib/dgram.js

```diff
@@ -344,6 +346,13 @@ Socket.prototype.send = function(buffer,
   if (typeof callback !== 'function')
     callback = undefined;
 
+  if (typeof address === 'function') {
+    callback = address;
+    address = undefined;
+  } else if (typeof address !== 'undefined' && typeof address !== 'string') {
```

Comment:
I'm not sure what you mean by 'acceptable,' but what I was saying is that

```js
address !== undefined && typeof address !== 'string'
```

and

```js
address != undefined && typeof address !== 'string'
```

will both evaluate to `true` for `address === null`.